static/js/components/SourceTable.jsx

```diff
@@ -443,6 +447,56 @@ const SourceTable = ({
     );
   };
 
+  const renderSpectrumExists = (dataIndex) => {
+    const source = sources[dataIndex];
+    return source.spectrum_exists ? (
+      <CheckIcon
+        size="small"
+        key={`${source.id}_spectrum_exists`}
+        color="primary"
+      />
+    ) : (
+      <ClearIcon
+        size="small"
+        key={`${source.id}_spectrum_exists`}
+        color="secondary"
+      />
+    );
+  };
+
+  const renderPeakMagnitude = (dataIndex) => {
+    const source = sources[dataIndex];
+    const peakPoint = source.photometry
+      .filter((point) => point.flux)
+      .sort((a, b) => a.flux - b.flux)[0];
+    return (
+      <Tooltip title={`${(mjdNow - peakPoint.mjd).toFixed(2)} days ago`}>
+        <div>{`${flux_to_mag(peakPoint.flux).toFixed(4)}`}</div>
```

Comment:
```suggestion
        <div>{`${flux_to_mag(peakPoint.flux, peakPoint.zp).toFixed(4)}`}</div>
```

Reply:
@dannygoldstein there doesn't seem to be a `zp` column in the Photometry table - should I add one?

Reply:
Or add it to the api response, more likely

Reply:
It's here: https://github.com/skyportal/skyportal/blob/master/skyportal/handlers/api/photometry.py#L123